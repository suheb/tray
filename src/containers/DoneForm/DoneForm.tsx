import React from "react";
import { Grid, Typography } from "@material-ui/core";
import DoneIcon from '@material-ui/icons/Done';


const DoneForm = () => {
  return (
    <Grid
      container
      justify="center"
      spacing={2}
    >
      <Grid
        item
        xs={2}
      >
        <DoneIcon fontSize="large" color="primary" />
      </Grid>

      <Grid
        item
        xs={12}
      >
        <Typography variant="subtitle1" color="primary" gutterBottom>
          Please verify your email address, you should have received an email from us already!
        </Typography>
      </Grid>
    </Grid>
  )
}

export default DoneForm;
