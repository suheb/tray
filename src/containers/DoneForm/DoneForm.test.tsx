import React from 'react';
import { render } from '@testing-library/react';

import DoneForm from './DoneForm';

describe("DoneForm", () => {
  it("renders without crashing", () => {
    const { asFragment } = render(<DoneForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
