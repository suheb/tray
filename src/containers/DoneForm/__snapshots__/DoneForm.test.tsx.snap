// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoneForm renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeLarge"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
        />
      </svg>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h6
        class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorPrimary MuiTypography-gutterBottom"
      >
        Please verify your email address, you should have received an email from us already!
      </h6>
    </div>
  </div>
</DocumentFragment>
`;
